modules/core/src/main/java/org/apache/ignite/internal/processors/job/GridJobProcessor.java

```diff
@@ -343,6 +357,15 @@ public GridJobProcessor(GridKernalContext ctx) {
 
                 return new ComputeJobView(e.getKey(), e.getValue(), state);
             });
+
+        if (!jobAlwaysActivate && ctx.collision().collisionSpi() instanceof PriorityQueueCollisionSpi) {
```

Comment:
Maybe extract `ctx.collision().collisionSpi()` into a variable?

Reply:
You're right, I can do that.